setup.py

```diff
@@ -26,7 +26,7 @@
 
 # What packages are required for this module to be executed?
 REQUIRED = [
-    # 'requests', 'maya', 'records',
+    'Django', 'djangorestframework'
```

Comment:
Why put this here instead of the pipfile?

Reply:
I assumed since we are still using `setup.py` to package, we need to require these. As they are the true base of the project. However, if it can be package and installed without these by all means I think we should remove it!

Reply:
Ah cool, nevermind– this makes sense.